src/scopeHandler.js

```diff
@@ -70,9 +75,15 @@ export function createScopeHandler(unsafeRec, safeGlobal, sloppyGlobals) {
         return undefined;
       }
 
-      // Properties of the global.
-      if (prop in target) {
-        return target[prop];
+      // Check endowments first.
+      if (prop in endowments) {
+        return endowments[prop];
+      }
+
+      // Get the value from the shadow. The target itself is an empty
+      // object that is only used to prevent a froxen eval property.
+      if (prop in safeGlobal) {
+        return safeGlobal[prop];
```

Comment:
It is weird but true that this can safely use square brackets rather than `Reflect.get`

Reply:
Performance is also much better FWIW.